README.md

```diff
@@ -76,34 +76,23 @@ Pronounced *semi*.
     bandwidth to remain available even after the volume's
     initial credits have been exhausted [2].
 
-    In Dec 2017, the ideal settings are as follows.
-
-      * c5.18xlarge with gp2 EBS
-
-        * number-of-slices >= 7
+    In Dec 2018, the ideal settings for a c5.9xlarge with gp2 EBS are
 
-        * slice-size >= 214GB
+        * number-of-slices = 3
 
-      * i3.16xlarge with gp2 EBS
+        * slice-size >= 334 GB   (increase this if you need more IOPS or space)
 
-        * number-of-slices >= 11
-
-        * slice-size >= 214GB
-
-  * Design question:  What if the instance where the RAID array
-    is to be mounted shouldn't have permissions to create new EBS
-    volumes?
 
 ## `s3mi raid array-name <block_device> <block_device> ...`
 
   Use RAID 0 with **instance NVME devices**.  For example,
 
-  `s3mi raid my_raid /dev/nvme{0..7}n1`
+  `s3mi raid my_raid /dev/nvme{1..8}n1`
```

Comment:
This is the new device numbering for nitro instances such as i3.metal.  The old numbering is still supported.